packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.types.ts

```diff
@@ -21,5 +21,7 @@ export type AvatarFaviconProps = Omit<
      * Optional prop to pass to the underlying img element
      * Useful for overriding the default alt text which is the dapp name
      */
-    imageProps?: ComponentProps<'img'>;
+    imageProps?: ComponentProps<'img'> & {
+      'data-testid'?: string;
```

Comment:
Unfortunately ComponentProps<'img'> doesn't include data attributes which we would usually expect to be accepted by any jsx component so we will have to specify it until we find a typed solution